source/Ship.cpp

```diff
@@ -2768,6 +2768,13 @@ const Planet *Ship::GetPlanet() const
 
 
 
+void Ship::OverrideCapture()
+{
+	isCapturable = true;
+}
```

Comment:
This should probably be a bit more generic, like just a method for setting `isCapturable`.

Reply:
Yeah taking a bool I see what you mean